tracer/build/_build/Build.GitHub.cs

```diff
@@ -86,6 +86,102 @@ partial class Build
+                    number: PullRequestNumber.Value);
+
+                // Fixes an issue (ambiguous argument) when we do git diff in the Action.
+                GitTasks.Git("fetch origin master:master", logOutput: false);
+
+                // This is a dumb implementation that just show the diff
+                // We could imagine getting the whole context with -U1000 and show the differences including parent name
+                // eg now we show -oldAttribute: oldValue, but we could show -tag.oldattribute: oldvalue
+                var changes = GitTasks.Git("diff master -- tracer/test/snapshots")
+                                   .Select(f => f.Text);
+
+
+                const int minFiles = 50;
+                var nbSnapshotsModified = changes.Count(f => f.Contains("@@ "));
+                if (nbSnapshotsModified < minFiles)
+                {
+                    // Dumb early exit, if we modify less than 50 files, we can review them manually
+                    // Also it's certainly an addition of snapshot tests, so may not make sense to print a summary.
+                    Console.WriteLine("${nbSnapshotsModified} snapshots modified. Not doing snapshots diff for PRs modifying less than ${minFiles}.");
+                    return;
+                }
+
+                const string unlinkedLinesExplicitor = "[...]";
+                var diffCounts = new Dictionary<string, int>();
+                StringBuilder diffsInFile = new();
+                var lastLineContainedAChange = false;
+                foreach (var line in changes)
+                {
+                    if (line.StartsWith("@@")) // new file
+                    {
+                        RecordChange(diffsInFile, diffCounts);
+                        lastLineContainedAChange = false;
+                        continue;
+                    }
+
+                    if (line.StartsWith("- ") || line.StartsWith("+ "))
+                    {
+                        diffsInFile.AppendLine(line);
+                        lastLineContainedAChange = true;
+                        continue;
+                    }
+
+                    if (lastLineContainedAChange)
+                    {
+                        diffsInFile.AppendLine(unlinkedLinesExplicitor);
+                        lastLineContainedAChange = false;
+                    }
+                }
+
+                RecordChange(diffsInFile, diffCounts);
+
+                var markdown = new StringBuilder();
+                markdown.AppendLine("## Snapshots difference summary").AppendLine();
+                markdown.AppendLine("The following differences have been observed in snapshots. So diff is simplistic, so please check some files anyway while we improve it.").AppendLine();
+
+                foreach (var diff in diffCounts)
+                {
+                    markdown.AppendLine($"{diff.Value} occurrences of : ");
+                    markdown.AppendLine("```");
```

Comment:
If this is printing in diff format (i.e. with `+`/`-` at the start) then you can add the language:

```suggestion
                    markdown.AppendLine("```diff");
```

Reply:
:wow: thanks